test/EFCore.PG.FunctionalTests/BulkUpdates/NorthwindBulkUpdatesNpgsqlTest.cs

```diff
@@ -551,78 +551,407 @@ SELECT 1
 """);
     }
 
-    public override async Task Update_where_constant(bool async)
+    public override async Task Update_Where_set_constant(bool async)
     {
-        await base.Update_where_constant(async);
+        await base.Update_Where_set_constant(async);
 
         AssertExecuteUpdateSql(
-            @"UPDATE ""Customers"" AS c
-    SET ""ContactName"" = 'Updated'
-WHERE c.""CustomerID"" LIKE 'F%'");
+"""
+UPDATE "Customers" AS c
+    SET "ContactName" = 'Updated'
+WHERE c."CustomerID" LIKE 'F%'
+""");
     }
 
-    public override async Task Update_where_parameter(bool async)
+    public override async Task Update_Where_parameter_set_constant(bool async)
     {
-        await base.Update_where_parameter(async);
+        await base.Update_Where_parameter_set_constant(async);
 
         AssertExecuteUpdateSql(
-            @"@__value_0='Abc'
+"""
+@__customer_0='ALFKI'
 
-UPDATE ""Customers"" AS c
-    SET ""ContactName"" = @__value_0
-WHERE c.""CustomerID"" LIKE 'F%'");
+UPDATE "Customers" AS c
+    SET "ContactName" = 'Updated'
+WHERE c."CustomerID" = @__customer_0
+""",
+            //
+"""
+@__customer_0='ALFKI'
+
+SELECT c."CustomerID", c."Address", c."City", c."CompanyName", c."ContactName", c."ContactTitle", c."Country", c."Fax", c."Phone", c."PostalCode", c."Region"
+FROM "Customers" AS c
+WHERE c."CustomerID" = @__customer_0
+""",
+            //
+"""
+SELECT c."CustomerID", c."Address", c."City", c."CompanyName", c."ContactName", c."ContactTitle", c."Country", c."Fax", c."Phone", c."PostalCode", c."Region"
+FROM "Customers" AS c
+WHERE FALSE
+""",
+            //
+"""
+UPDATE "Customers" AS c
+    SET "ContactName" = 'Updated'
+WHERE FALSE
+""");
     }
 
-        public override async Task Update_where_using_property_plus_constant(bool async)
+    public override async Task Update_Where_set_parameter(bool async)
     {
-        await base.Update_where_using_property_plus_constant(async);
+        await base.Update_Where_set_parameter(async);
 
         AssertExecuteUpdateSql(
-            @"UPDATE ""Customers"" AS c
-    SET ""ContactName"" = COALESCE(c.""ContactName"", '') || 'Abc'
-WHERE c.""CustomerID"" LIKE 'F%'");
+"""
+@__value_0='Abc'
+
+UPDATE "Customers" AS c
+    SET "ContactName" = @__value_0
+WHERE c."CustomerID" LIKE 'F%'
+""");
+    }
+
+    [ConditionalTheory(Skip = "Non-deterministic, no ordering")]
```

Comment:
/cc @smitpatel

Reply:
There are 3 tests without ordering. It is fine to skip them. There are tests which provides order by with skip/take so it is covered. SqlServer allows Take without ordering to be put on top level so added those tests.

Reply:
OK. Do you want to maybe move these tests to SQL Server, since they rely on non-deterministic SQL Server ordering in order to pass? This would save the trouble for other providers implementing these from figuring out what's wrong, that they need to skip, etc.

Reply:
BTW what's the third :) I'm only seeing 2 ordering-related failures :)